src/pytest_benchmark/plugin.py

```diff
@@ -424,7 +424,7 @@ def pytest_runtest_setup(item):
         for name in marker.kwargs:
             if name not in (
                     "max_time", "min_rounds", "min_time", "timer", "group", "disable_gc", "warmup",
-                    "warmup_iterations", "calibration_precision", "cprofile"):
+                    "warmup_iterations", "calibration_precision", "cprofile", "baseline"):
```

Comment:
Now sure how but we should have some way to validate that there is only 1 baseline per group. It doesn't make sense to have 2 baselines right? Lets not let users wonder why stuff doesn't work as expected (the annoying silent failure).

Reply:
The current implementation marks all results as possible baselines by default and only excludes the ones marked as `baseline=False`. If there is more than one `baseline=True` benchmark available it will choose the one with the lowest value/highest score. This perfectly integrates with the existing behaviour and means that I don't have to choose the baseline value selected for all times (as performance may differ between systems, etc). The included docs actually mention this. As you mentioned there cannot be *baselines* when the output is rendered, but there can be more than one *potential* baseline scores.

Unless you have strong feelings on this, I'd like to keep it this way for extra flexibility. The wording could be improved however: maybe something along the lines of `potential_baseline`, but shorter?